ts/Maps/MapView.ts

```diff
@@ -164,21 +164,24 @@ class MapView {
                 }
 
                 // Panning rotates the globe
```

Comment:
This comment should be further down, before line 172. Otherwise the context is lost.